lib/themes/dosomething/paraneue_dosomething/js/utilities/Globe.js

```diff
@@ -0,0 +1,12 @@
+import $ from 'jquery';
+import setting from './Setting';
+
+function init() {
+  const globeUpdateUrl = setting('dosomethingSetting.globeUrl');
```

Comment:
@sergii-tkachenko It's fine, we're running all our scripts through [Babel](https://babeljs.io) to transpile to ES5.